### DISCUSSION TITLE
ImportError: cannot import name 'registry'

### DISCUSSION THREAD

**Discussion by Sarah-air** [stellargraph/stellargraph#1804] [General] [answered]
https://stellargraph.readthedocs.io/en/stable/demos/node-classification/gcn-node-classification.html

  from pandas.core.dtypes.base import ExtensionDtype, registry
ImportError: cannot import name 'registry'

**Comment by huonw** [comment]
Hi, thanks for asking a question. We can help you better if you give more information. For example:

- package versions with `pip freeze` (to understand if this is a version-mismatch problem)
- the full stack trace of the error (to tell us where this problem occurs)

**Comment by Sarah-air** [comment, marked as answer]
Thank you for your reply! I have solved this problem by updating the related package.





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "stellargraph/stellargraph"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年9月21日(星期一) 上午7:22
收件人:&nbsp;"stellargraph/stellargraph"<stellargraph@noreply.github.com&gt;;
抄送:&nbsp;"528497479"<528497479@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [stellargraph/stellargraph] ImportError: cannot import name 'registry' (#1804)





 
Hi, thanks for asking a question. We can help you better if you give more information. For example:
  
package versions with pip freeze (to understand if this is a version-mismatch problem)
 
the full stack trace of the error (to tell us where this problem occurs)
